atd-vze/src/Components/DataTable.js

```diff
@@ -149,66 +145,89 @@ const DataTable = ({
                       };
 
                       const fieldValueDisplay = renderLookupDescString();
+                      const canClickToEdit =
+                        !isReadOnlyUser &&
+                        fieldConfigObject.editable &&
+                        !isEditing;
 
                       return (
-                        <tr key={i}>
-                          <td>
+                        <tr
+                          key={i}
+                          onClick={() => canClickToEdit && setEditField(field)}
+                          style={{
+                            cursor: canClickToEdit ? "pointer" : "auto",
+                          }}
+                        >
+                          <td className="align-middle">
                             <strong>{fieldLabel}</strong>
                           </td>
-                          <td>
+                          <td colSpan={isEditing ? 2 : 1}>
                             {isEditing ? (
                               <form
                                 onSubmit={e =>
                                   handleFieldUpdate(e, section.fields, field)
                                 }
                               >
-                                {fieldUiType === "select" && (
-                                  <Input
-                                    name={field}
-                                    id={field}
-                                    onChange={e => handleInputChange(e)}
-                                    defaultValue={fieldValue}
-                                    type="select"
-                                  >
-                                    {selectOptions.map(option => (
-                                      <option
-                                        value={option[`${lookupPrefix}_id`]}
+                                <div className="d-flex">
+                                  <div className="flex-grow-1">
+                                    {fieldUiType === "select" && (
+                                      <Input
+                                        autoFocus
+                                        name={field}
+                                        id={field}
+                                        onChange={e => handleInputChange(e)}
+                                        defaultValue={fieldValue}
+                                        type="select"
                                       >
-                                        {option[`${lookupPrefix}_desc`]}
-                                      </option>
-                                    ))}
-                                  </Input>
-                                )}
-                                {fieldUiType === "text" && (
-                                  <input
```

Comment:
Switching from a plain old `<input>` to the bootstrapped `<Input>` component. This matches the `select` component above.